walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/Event.java

```diff
@@ -53,6 +54,12 @@ public Event(String title, TimeRange when, Collection<String> attendees) {
     this.attendees.addAll(attendees);
   }
 
+  public static final Comparator<Event> ORDER_BY_START = new Comparator<Event>() {
```

Comment:
I don't think that methods are conventionally treated as constants. If you have a source that says otherwise though I'm happy to be proved wrong.